### ISSUE TITLE
Please provide basic documentation

### ISSUE THREAD

**Issue by gerroon** [nvllsvm/jellysub#1]
Hi

It would be nice if there is a doc that explains how to use this. The cli arguments are not much helpful.

**Comment by DJWhitegold** [comment]
Yeah installation and usage instructions for windows would be very helpful. This project sounds like a great way to offer flexibility and choice for android apps but I have no idea how to Install or use it.

**Comment by Odroid-Dude** [comment]
Likewise, I think I have it running correctly but having no luck getting play:Sub (iOS) to connect to my Jellyfin server.
Have entered jellyfin.mydomain.com:4040 Is this correct?
